lisa/features/startstop.py

```diff
@@ -48,3 +48,6 @@ def restart(self, wait: bool = True) -> None:
         self._log.info("restarting")
         self._restart(wait=wait)
         self._node.close()
+
+    def status(self) -> str:
```

Comment:
If it's a method call it like `get_status`, or it can be a property called `status`.